internal/v1/handler.go

```diff
@@ -359,21 +367,21 @@ func (h *Handlers) GetComposeMetadata(ctx echo.Context, composeId uuid.UUID) err
 }
 
 // return an error if the user does not have the composeId associated to its OrgID in the DB, nil otherwise
-func (h *Handlers) canUserAccessComposeId(ctx echo.Context, composeId uuid.UUID) error {
+func (h *Handlers) canUserAccessComposeId(ctx echo.Context, composeId uuid.UUID) (*db.ComposeEntry, error) {
```

Comment:
`canUserAccessComposeId` no longer describes well what the method does - it now returns the compose, instead of just checking its access rights. Also, the comment on the line above must be changed.